sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -425,6 +425,24 @@ def _add_argparse_args(cls, parser):
         help='Whether to enable streaming mode.')
 
 
+class CrossLanguageOptions(PipelineOptions):
+  @classmethod
```

Comment:
Hmm, I thought I understood this but may be I didn't :).

Is this supposed to get picked by expansion services automatically started by IOs (for example Kafka) ? That doesn't seem to be working.

Reply:
I was able to use this to override the jar used in SqlTransform. The one gotcha was that the artifact name had to exactly match the one in BeamJarExpansionService (in my case I had omitted the leading ":").

I did run into another issue though. When I set this parameter and run on Dataflow the job fails because workers are unable to parse this arg. It seems somewhere along the line it's translate to a python dictionary like `{':sdks:java:..': '/blah/blah'}` and no longer valid json.